src/ServicePulse.Host/vue/src/components/monitoring/EndpointListRow.vue

```diff
@@ -36,8 +36,10 @@ const criticalTimeGraphDuration = computed(() => formatGraphDuration(endpoint.va
         <RouterLink :to="routeLinks.monitoring.endpointDetails.link(endpoint.name, historyPeriod.pVal)" class="cursorpointer" v-tooltip :title="endpoint.name">
           {{ shortName }}
         </RouterLink>
-        <span class="endpoint-count" v-if="endpoint.connectedCount || endpoint.disconnectedCount" v-tooltip :title="`Endpoint instance(s):${endpoint.connectedCount}`">({{ endpoint.connectedCount }})</span>
       </div>
+      <span class="endpoint-count" v-if="endpoint.connectedCount || endpoint.disconnectedCount" v-tooltip title="Endpoint instance(s): (connected/total)">
```

Comment:
Does this mean if both are zero we will display nothing?

Reply:
i believe in that scenario we wouldn't show the row at all